scout/server/blueprints/cases/views.py

```diff
@@ -777,10 +778,23 @@ def delivery_report(institute_id, case_name):
     else:
         delivery_report = case_obj['delivery_report']
 
+    format = request.args.get('format','html')
+    if format == 'pdf':
+        try: # file could not be available
+            html_file = open(delivery_report, 'r')
+            source_code = html_file.read()
+            # remove image, since it is problematic to render it in the PDF version
+            source_code=re.sub('<img class=.*?alt="SWEDAC logo">','',source_code, flags=re.DOTALL)
+            return render_pdf(HTML(string=source_code), download_filename=case_obj['display_name']+'_'+datetime.datetime.now().strftime("%Y-%m-%d")+'_scout_delivery.pdf')
+        except Exception as ex:
+            flash('An error occurred while download {} -- {}'.format(delivery_report, ex), 'warning')
```

Comment:
"during download of" or "while downloading". 😄

Reply:
Haha yes! :)